### PR TITLE
Added options for filtering messages

### PR DESCRIPTION
**Pull request by joaquin-corchero**
Added options for filtering messages

Updated README.md

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "github-release-notes",
-  "version": "0.3.3",
+  "version": "0.3.4",
```
Comment (alexcanessa) [inline package.json]: Since it is a new feature, the version should be `0.4.0`

File: src/index.js [hunk 2]
```diff
@@ -69,11 +69,19 @@ function prepareRelease(gren, tagName, commitMessages) {
...
       .filter(function (message) {
-         return !message.match(/^merge/i);
+          switch(gren.includeMessages)
+          {
+              case 'MergePull':
+                return message.match(/^merge/i);
+              case "Commits":
+                return !message.match(/^merge/i);
+              default:
+                return true;
```
Comment (alexcanessa) [inline src/index.js]: I prefer to keep the default to `Commits`, to avoid a breaking change.

File: README.md [hunk 3]
```diff
@@ -30,4 +30,5 @@ There are optional arguments such as:
 
 - `--draft=true` To set the release as a draft. Default: `false`
 - `--prerelease=true` To set the release as a prerelease. Default: `false`
-- `--prefix=v` Add a prefix to the tag version `e.g. v1.0.1`
\ No newline at end of file
+- `--prefix=v` Add a prefix to the tag version `e.g. v1.0.1`
+- `--includeMessages=MergePull/Commits/All` used to filter the messages added to the release notes. Default: `Commits`
```
Comment (alexcanessa) [inline README.md]: - Can we use `--include-messages` as camel case isn't that common in CLI programs.
- I'd rather the options to be: `merges/commits/all`

Comment (joaquin-corchero) [inline README.md]: Changed options to your suggested and the parameter to lowercase

**Comment by alexcanessa** [comment]
Thanks for contributing.

This closes #5

**Comment by alexcanessa** [comment]
@all-contributors add @joaquin-corchero for code

**Pull request merged**